File: ai_generated_photos/v2/get/iOS SDK/AIGeneratedPhotos.swift
import Amadeus

var amadeus: Amadeus = Amadeus(
    client_id: "YOUR_API_ID",
    client_secret: "YOUR_API_SECRET"
)

/// Generates a fake mountain image
amadeus.media.files.generatedPhotos.get(params: ["category": "MOUNTAIN"],
                                        onCompletion: {
                                            response, error in
                                            if error == nil {
                                                print(response!.data)
                                            }
                                        })

File: ai_generated_photos/v2/get/Node SDK/ai_generated_photos.js
var Amadeus = require("amadeus");
var amadeus = new Amadeus({
  clientId: 'YOUR_API_KEY',
  clientSecret: 'YOUR_API_SECRET'
});

// Generates a photo with mountain
amadeus.media.files.generatedPhotos.get({
  category: 'MOUNTAIN'
}).then(function (response) {
  console.log(response);
}).catch(function (response) {
  console.error(response);
});